packages/cli/package.json

```diff
@@ -59,7 +59,8 @@
     "levelup": "^4.4.0",
     "morgan": "^1.10.0",
     "nft-did-resolver": "^1.0.0",
-    "pkh-did-resolver": "^0.3.8-rc.0",
+    "picocolors": "^1.0.0",
+    "pkh-did-resolver": "^0.3.3",
```

Comment:
@Sterahi mind I ask you, please, why pkh-did-resolver version has changed?

Reply:
I have no idea, I've reset it to it's previous iteration though!